dts/bindings/serial/silabs,eusart-uart.yaml

```diff
@@ -0,0 +1,24 @@
+description: Silabs EUSART UART
+
+compatible: "silabs,eusart-uart"
+
+include: [uart-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  current-speed:
+    default: 115200
+
+  parity:
+    default: "none"
+
+  stop-bits:
+    default: "1"
+
+  data-bits:
+    default: 8
```

Comment:
As per the feedback in the other PR where defaults are added to existing drivers, I believe these should also get a `description:` field.

Reply:
done